packages/fxa-settings/src/components/FlowContainer/index.tsx

```diff
@@ -17,14 +18,15 @@ type FlowContainerProps = {
 };
 
 export const FlowContainer = ({
+  width = 'lg',
```

Comment:
I'm curious why you needed to do this here

Reply:
I replaced the default width class so that we can pass different size options into FlowContainer, setting `lg` as the default since other components likely depend on that

Reply:
I'm a bit confused by this. Why would we want the width of this page to be different from the width of all the other pages?